src/common/cli.ts

```diff
@@ -34,8 +41,19 @@ export const waitForProcessingResult = async (
     let times = 20
     while (times-- > 0) {
       const statuses = await statusCommand(messageIds)
-      console.log(`Processing status: ${statuses} (${20 - times}/20)`)
-      if (!statuses.some(s => s !== "DONE")) {
+      const states = statuses.map(s => s.importStatus)
+      console.log(`Processing status: ${states.join(',')} (${20 - times}/20)`)
+      if (allImportsAreFinished(statuses)) {
+        statuses.forEach(s => {
+          if (s.error !== null) {
+            console.log("\nError in request " + s.messageId + ": " + s.error)
```

Comment:
chore: Konkatenoinnin sijaan voisi käyttää interpolointia, sama alemmissa
```suggestion
            console.log(`\nError in request ${s.messageId}: ${s.error}`)
```